docker-fedora/Dockerfile

```diff
@@ -26,5 +28,8 @@ COPY config.json.template service.template tmpfiles.template /exports/
 COPY daemon.json /exports/hostfs/etc/docker/container-daemon.json
 # https://github.com/rhatdan/oci-umount/issues/2
 RUN cp /etc/oci-umount.conf /exports/hostfs/etc
+# Copy config if available
+RUN (test -e /etc/oci-umount.conf && cp /etc/oci-umount.conf /exports/hostfs/etc) || true
```

Comment:
Shouldn't the line above this be removed?

Also this change is already in f25, but it *shouldn't* be an issue for later versions (as the file will be present)